components/search/NotFound/index.tsx

```diff
@@ -1,29 +1,33 @@
-import NotFoundImage from 'public/img/notFound.svg';
+import NotFoundImage from 'public/img/notFound.png';
 import Image from 'next/image';
 import { PropsType } from './NotFound.type';
 import Link from 'next/link';
 
 function NotFound(props: PropsType) {
   const { query } = props;
   return (
-    <div className="container px-25 md:px-0 py-30 md:py-[120px] flex flex-col items-center">
-      <Image
-        src={NotFoundImage}
-        className="h-[148px] md:h-[318px]"
-        alt="Not Found"
-      />
-      <div className="text-16  md:text-45 pt-[45px] md:pt-[100px] text-center text-primary-500 font-semibold">
-        Oops! We couldn't find what you were looking for.
+    <div className="px-25 md:px-0 py-[75px] md:py-[120px] flex flex-col items-center max-w-3xl">
+      <div className="w-full relative">
+        <Image
+          src={NotFoundImage}
+          className="max-w-[250px] md:max-w-max mx-auto"
+          alt="Not Found"
+        />
       </div>
-      <div className="text-14 text-center md:text-20 text-neutral-800 pt-10">
+      <div className="text-18 md:text-45 mt-30 md:mt-50 text-center text-primary-500 font-semibold">
+        Oops! We couldn't find what you are looking for
+      </div>
+      <div className="text-14 text-center md:text-20 text-neutral-800 mt-10">
         {`The search string you entered was: ${query || ''}`}
+        <br />
+        This is an invalid search string.
       </div>
       {/* <div className="text-14 text-center md:text-20 text-neutral-800">
         This is an invalid search string.
       </div> */}
       <Link
         href="/"
-        className="text-baseForeground bg-primary-600 py-10 md:py-15 px-20 mt-50 text-center rounded-full w-[224px] text-14 md:text-18 font-medium">
```

Comment:
The height, width and padding of this button in mobile and desktop mode do not match with Figma.

Reply:
Tnx, I tried to improve it but the design has some problems. For example, in Figma in large screen the height of button is 48px with 15px vertical padding and 10px content, but the line height of the text inside it is 26px.